docs/abi-spec.rst

```diff
@@ -113,6 +113,9 @@ them.
 +-------------------------------+-----------------------------------------------------------------------------+
 |:ref:`enum<enums>`             |``uint8``                                                                    |
 +-------------------------------+-----------------------------------------------------------------------------+
+|:ref:`user defined value types |same as its underlying value type                                            |
```

Comment:
"same as" sounds like it is not exactly the underlying value type

Reply:
Changed it to "exactly as its underlying value type"